profiles/DishaKD.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://avatars.githubusercontent.com/u/87950341?v=4",
```

Comment:
this should be a screenshot of you profile not the profile picture itself

Reply:
Fixed that issue